cfgov/unprocessed/apps/paying-for-college/js/dispatchers/update-models.js

```diff
@@ -6,6 +6,7 @@ import { financialModel } from '../models/financial-model.js';
 import { financialView } from '../views/financial-view.js';
 import { getStateByCode } from '../util/other-utils.js';
 import { getSchoolData } from '../dispatchers/get-api-values.js';
+import { navigationView } from '../views/navigation-view.js';
```

Comment:
Is this import necessary?

Reply:
Nope! I think it's a remnant of the "skip the intro page" work I was doing. I've removed it.